scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py

```diff
@@ -1467,45 +1373,7 @@ def ZH_Verwaltungsgericht(decision: Union[bs4.BeautifulSoup, str], namespace: di
 
     section_markers = prepare_section_markers(all_section_markers, namespace)
 
-    def get_paragraphs(soup):
-        """
-        Get Paragraphs in the decision
-        :param soup:    the decision parsed by bs4 
-        :return:        a list of paragraphs
-        """
-        # sometimes the div with the content is called WordSection1
-        divs = soup.find_all("div", class_="WordSection1")
-        # sometimes the div with the content is called Section1
-        if len(divs) == 0:
-            divs = soup.find_all("div", class_="Section1")
-        # we expect maximally two divs with class WordSection1
-        assert (len(divs) <= 2), "Found more than two divs with class WordSection1"
-        assert (len(divs) > 0), "Found no div, " + str(namespace['html_url'])
-
-        paragraphs = []
-        heading, paragraph = None, None
-        for element in divs[0]:
-            if isinstance(element, bs4.element.Tag):
-                text = str(element.string)
-                # This is a hack to also get tags which contain other tags such as links to BGEs
-                if text.strip() == 'None':
-                    text = element.get_text()
-                # get numerated titles such as 1. or A.
-                if "." in text and len(text) < 5:
-                    heading = text  # set heading for the next paragraph
-                else:
-                    if heading is not None:  # if we have a heading
-                        paragraph = heading + " " + text  # add heading to text of the next paragraph
-                    else:
-                        paragraph = text
-                    heading = None  # reset heading
-                # only clean and append non-empty paragraphs
-                if paragraph not in ['', ' ', None]:
-                    paragraph = clean_text(paragraph)
-                    paragraphs.append(paragraph)
-        return paragraphs
-
-    paragraphs = get_paragraphs(decision)
+    paragraphs = get_paragraphs_unified(decision)
```

Comment:
did you test if this works equally well?

Reply:
Yes I tested it. Also, the pattern extractor was always using the get_paragraphs_unified method to extract the patterns which means that it works.

Reply:
Great, thanks